framework/gui.h

```diff
@@ -407,6 +417,10 @@ class Gui
 
 	void set_subpass(const uint32_t subpass);
 
+	VkSampler get_sampler() const;
+
+	VkImageView get_font_image_view() const;
+
```

Comment:
Since `sampler` and `image_view` are defined as `std::unique_ptr`'s this would mean that it's either returned by `std::move` - not a good idea to change ownership of a pointer of internal class member. Or alternatively, in my opinion even worse to return raw pointer. It could be also defined as `std::shared_ptr` and this means a design change.

Reply:
I think, returning a `vkb::core::Sampler const &` and a `vkb::core::ImageView const &` would be right.
As as they both don't support copy construction or copy assignment, you would not be able to accidentally assign or copy them.